pex/resolve/lockfile/json_codec.py

```diff
@@ -297,3 +352,8 @@ def as_json_data(lockfile):
             for locked_resolve in lockfile.locked_resolves
         ],
     }
+    if path_mappings.mappings:
+        data["path_mappings"] = {
+            path_mapping.name: path_mapping.description for path_mapping in path_mappings.mappings
+        }
```

Comment:
Generally you are always defining keys, and using empty values when relevant. Why conditionally register this key?

Reply:
Its a bit different in that a very large percentage of locks will never use path mappings, but I guess that may not be any reason at all - I'll take a look.

Reply:
Yeah, it was for some sort of misguided symmetry with the read side I think. On the read side I have to deal with the key not being there to be compatible with old locks. Symmetry is just beauty though and that's in the eye of the beholder and pretty much garbage for basing a decision on. Fixed!